manticore/core/smtlib/solver.py

```diff
@@ -211,10 +243,19 @@ def stop(self):
             # No need to wait for termination, zombies avoided.
         self._proc = None
 
-    def __readline_and_count(self):
+    def __readline_and_count(self, wait):
         assert self._proc
         assert self._proc.stdout
-        buf = self._proc.stdout.readline()  # No timeout enforced here
+        buf = None
+
+        while buf is None:
+            try:
+                buf = self._proc.stdout.readline()
```

Comment:
busy waiting?
I wonder when would the original readline wait forever.

Reply:
Fixed.